breadcrumb/subscribe-to-location-changed/readme.md

```diff
@@ -0,0 +1,45 @@
+# Breadcrumb - Subscribe to NavigationManager's LocationChanged
```

Comment:
```suggestion
# Breadcrumb - Automatically Generate Items Based on URL
```

Make the title more catchy - what it solves for the customer, not the exact code (there is room for that in the body). Consider expanding on the pain point this solves as the first paragraph (short description of the problem, situation and goal), and then go to the technical details in the next paragraphs.